docker-compose.yaml

```diff
@@ -0,0 +1,37 @@
+# This compose setup is only meant for local development of Misago itself
+# This is not for running your Misago site in docker
+version: "3.0"
+services:
+  postgres:
+    image: postgres:9
+    environment:
+      - POSTGRES_USER=misago
+      - POSTGRES_PASSWORD=misago
+      - POSTGRES_DB=misago
+    volumes:
+      # To keep things simple we map the postgres data into a docker volume.
+      # When re-creating the database you need to delete the volume with `docker volume rm`.
+      # You can also map the path to the host if this is more convenient
+      - postgres:/var/lib/postgresql/data
```

Comment:
There's no need for postgresql volume in dev IMHO, we already have tools that let dev to quickly fill the database with mock data if they wish so.

Reply:
That's a good point. We can just let the data die with the container.